sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -1,37 +1,293 @@
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[],
+            tables_to_keep=[
+                MockFeatureView(name=self.collection_to_write, schema=schema)
+            ],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+
+        assert f"Collection {self.collection_to_write} already exists." in caplog.text
+
+    def test_milvus_update_collection_with_unsupported_schema(
+        self, repo_config, caplog, milvus_online_setup
+    ):
+
+        # Checking to see if Milvus will raise an exception since primary key of type FLOAT_VECTOR is unsupported.
+        with pytest.raises(Exception):
+            # Creating a common schema for collection
+            schema = CollectionSchema(
+                fields=[
+                    FieldSchema(
+                        "int64", DataType.INT64, description="int64", is_primary=False
+                    ),
+                    FieldSchema(
+                        "float_vector", DataType.FLOAT_VECTOR, is_primary=True, dim=128
+                    ),
+                ]
+            )
+
+            MilvusOnlineStore().update(
+                config=repo_config,
+                tables_to_delete=[],
+                tables_to_keep=[
+                    MockFeatureView(name=self.collection_to_write, schema=schema)
+                ],
+                entities_to_delete=None,
+                entities_to_keep=None,
+                partial=None,
+            )
+
+    def test_milvus_update_delete_collection(
+        self, repo_config, caplog, milvus_online_setup
+    ):
+
+        # Creating a common schema for collection
+        schema = CollectionSchema(
+            fields=[
+                FieldSchema(
+                    "int64", DataType.INT64, description="int64", is_primary=True
+                ),
+                FieldSchema(
+                    "float_vector", DataType.FLOAT_VECTOR, is_primary=False, dim=128
+                ),
+            ]
+        )
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[],
+            tables_to_keep=[
+                MockFeatureView(name=self.collection_to_delete, schema=schema)
+            ],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[
+                MockFeatureView(name=self.collection_to_delete, schema=None)
+            ],
+            tables_to_keep=[],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+        assert utility.has_collection(self.collection_to_delete) is False
+        assert len(utility.list_collections()) == 0
+        assert (
+            f"Collection {self.collection_to_delete} has been deleted successfully."
```

Comment:
do not assert log-statements. If the wording changes one has to go and update the tests.